pkg/scheduler/testing/framework_helpers.go

```diff
@@ -65,22 +81,29 @@ func RegisterPluginAsExtensions(pluginName string, weight int32, pluginNewFunc f
 func getPluginSetByExtension(plugins *schedulerapi.Plugins, extension string) *schedulerapi.PluginSet {
 	switch extension {
 	case "QueueSort":
-		return plugins.QueueSort
+		return initializeIfNeeded(&plugins.QueueSort)
```

Comment:
As discussed offline with @Huang-Wei, I'm hiding the double pointer details in this function, instead of RegisterPluginAsExtensions

Reply:
I don't think it was a big deal because it was already a local function, but this is ok as well.